src/test/java/uk/gov/justice/hmpps/prison/service/DataLoaderRepository.kt

```diff
@@ -36,24 +35,23 @@ class DataLoaderRepository(
   val institutionAreaRepository: ReferenceCodeRepository<InstitutionArea>,
   val teamCategoryRepository: ReferenceCodeRepository<TeamCategory>,
   val jdbcTemplate: JdbcTemplate,
-) {
+)
+
+@Service
+class DataLoaderTransaction {
```

Comment:
It was always confusing have this transactional methods on the loader itself, so create a special class that used purely to create a transaction when loading data